model.py

```diff
@@ -2239,13 +2239,9 @@ class Edition(Base):
     primary_identifier_id = Column(
         Integer, ForeignKey('identifiers.id'), index=True)
 
-    # A Edition may be associated with a single Work.
-    work_id = Column(Integer, ForeignKey('works.id'), index=True)
+    # An Edition may be the presentation edition for a single Work.
+    work = relationship("Work", uselist=False, backref="presentation_edition")
```

Comment:
I think it's okay to just call it `work` with the knowledge that `Edition.work` can now be None if the Edition isn't a presentation edition.